Libraries/Image/RCTImageLoader.m

```diff
@@ -377,9 +371,26 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
                                                    userInfo:nil], nil);
           return;
         }
+        
+        NSString* responseDate = ((NSHTTPURLResponse *)response).allHeaderFields[@"Date"];
```

Comment:
nit: our formatting guideline is `NSString *responseDate`. Could you apply that consistently everywhere?